doc/dev/ROADMAPS_AND_RELEASES.md

```diff
@@ -0,0 +1,29 @@
+# Roadmaps and release cycles
+
+F3D is a community driven project and we try to provide as much information as we can about the upcoming features
+and release expectations. Please keep in mind while we may try to fulfill what was planned, some features may not be available
+as planned and releases can be delayed.
+
+Roadmaps are available on their dedicated issue with their associated project board and milestone.
+They are bound to evolve over time.
+
+If you want to get involved, please feel free to comment on the issues and contribute to the issues that matters to you!
+
+## 2.4.0
+
+Our next minor release is planned for April 2024
+ - [Issue](https://github.com/f3d-app/f3d/issues/1242)
+ - [Milestone](https://github.com/f3d-app/f3d/milestone/9)
+ - [Board](https://github.com/orgs/f3d-app/projects/2/views/11?sliceBy%5Bvalue%5D=2.4.0)
+
+## 3.0.0
+
+Out next major release is planned for October 2024
+ - [Issue](https://github.com/f3d-app/f3d/issues/1243)
+ - [Milestone](https://github.com/f3d-app/f3d/milestone/8)
+ - [Board](https://github.com/orgs/f3d-app/projects/2/views/11?sliceBy%5Bvalue%5D=3.0.0)
+
+## Release cycle
+
+F3D release cycle is one minor release every three months, in January, April, July and October.
```

Comment:
I'm not against it, and actually I think it's a good idea to have more frequent releases, but that's also time consuming, so 3 months might be quite short.  
Anyway, let's try and see